flake.nix

```diff
@@ -1,40 +1,20 @@
 {
   inputs.nixpkgs.url = "github:NixOS/nixpkgs?ref=nixpkgs-unstable";
   outputs = { self, nixpkgs }: {
-    nixosModules.default = { config, lib, ... }:
-      let cfg = config.services.prometheus.exporters.cgroup; in {
-        options.services.prometheus.exporters.cgroup = {
-          enable = lib.mkEnableOption "cgroup-exporter";
-          listenAddress = lib.mkOption {
-            type = lib.types.str;
-            default = "[::]";
-            description = "Address to listen on";
-          };
-          port = lib.mkOption {
-            type = lib.types.int;
-            default = 13232;
-            description = "Port to listen on";
-          };
-        };
-        config = lib.mkIf cfg.enable {
-          systemd.services.cgroup-exporter = {
-            description = "cgroup-exporter";
-            wantedBy = [ "multi-user.target" ];
-            serviceConfig = {
-              Type = "simple";
-              ExecStart = "${self.packages.x86_64-linux.default}/bin/cgroup-exporter -listen-address ${cfg.listenAddress}:${toString cfg.port}";
-              Restart = "always";
-            };
-          };
-        };
-      };
+    nixosModules.default = { config, lib, ... }: {
```

Comment:
Can't this just be `nixosModules.default = ./nix/module.nix` ?

Not 100% sure actually. Can also find out after merging this if you don't feel like messing around with it.

Reply:
No because ./nix/module.nix's default package assume you used the overlay.